src/main/kotlin/uk/gov/justice/digital/hmpps/hmppschallengesupportinterventionplanapi/config/HmppsChallengeSupportInterventionPlanApiExceptionHandler.kt

```diff
@@ -192,6 +193,7 @@ class HmppsChallengeSupportInterventionPlanApiExceptionHandler {
         status = BAD_REQUEST,
         userMessage = "Invalid request: ${e.message}",
         developerMessage = e.message,
+        moreInfo = if (e is MissingComponentException) e.recordUuid.toString() else null,
```

Comment:
would it be better if the additional info goes into e.message, so we can the uuid in user message and dev message?